src/Microsoft.Sbom.Api/Microsoft.Sbom.Api.csproj

```diff
@@ -24,17 +24,11 @@
         <PackageReference Include="Serilog.Extensions.Hosting" />
         <PackageReference Include="Serilog.Sinks.Console" />
         <PackageReference Include="Spectre.Console.Cli" />
-        <PackageReference Include="System.IO.FileSystem.AccessControl" />
```

Comment:
What is the reason for the changes in this file?

Reply:
When I built locally, I got an error that some package references might not be needed, and it pointed here. I deleted it, and it still built, so it seems to have been right.